Library/Formula/stunnel.rb

```diff
@@ -65,9 +66,31 @@ def caveats
         #{etc}/stunnel/stunnel.pem
 
       This certificate will be used by default unless a config file says otherwise!
+      Stunnel will refuse to load the sample configuration file if left unedited.
 
       In your stunnel configuration, specify a SSL certificate with
       the "cert =" option for each service.
     EOS
   end
+
+  test do
+    (testpath/"tstunnel.conf").write <<-EOS.undent
+      cert = #{etc}/stunnel/stunnel.pem
+
+      setuid = nobody
+      setgid = nobody
+
+      [pop3s]
+      accept  = 995
+      connect = 110
+
+      [imaps]
+      accept  = 993
+      connect = 143
+    EOS
+
+    # If words "Configuration successful" appear in `brew test --verbose` then test = success.
+    # The exit 1 error is because to bind ports already in use you tend to require `sudo`.
+    system "(#{bin}/stunnel #{testpath}/tstunnel.conf; 2>&1)"
```

Comment:
Use `system_output` and `assert_match` here and it'll stop whining about system. You can set the exit code and match on `stderr` then too.